bad/belief/public_belief_global.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, hanabi_env: HanabiEnv, constants: Constants,
         self.hanabi_env = hanabi_env
 
         super().__init__(self.init(constants, self.rem_cards,
-                         self.hint_matrix, self.likelihood))
```

Comment:
Fehler liegt in der Deklaration ist ist likelihood. 
Muss in self.likelihood geändert werden